### ISSUE TITLE
RapiD Tag Removal Bug

### ISSUE THREAD

**Issue by bsidhuFB** [facebookincubator/RapiD#560]
### Description

Testing out RapiD. I removed a building tag from a building. Afterwards, the map froze, the building is permanently highlighted while not having the building tag. The map still moves around, and am able to select and move nodes, but the zoom level is frozen and does not show any changes.

### Version

2.0-alpha

### What browser are you seeing the problem on? What version are you running?

Chrome v104.0

### Steps to reproduce

1. Select buildings
2. remove a building tag
3. try clicking away from building/moving imagery

![Screen Shot 2022-08-17 at 5 21 45 PM](https://user-images.githubusercontent.com/111467504/185265951-bb17cada-65f7-4b34-9006-43d0350c42f7.png)

![Screen Shot 2022-08-17 at 5 22 05 PM](https://user-images.githubusercontent.com/111467504/185265946-5c454a5e-3c9a-431f-90c1-edfa28a9f891.png)

### The browser URL at the time you encountered the bug

https://mapwith.ai/rapid-v2-alpha#background=Bing&datasets=fbRoads,msBuildings&disable_features=boundaries&map=19.26/37.57662/-122.03116

**Comment by Bonkles** [comment]
This is a good find. I inspected things using the console and was able to reproduce: 

```

    PixiJS 6.4.2 - ✰ WebGL 2 ✰      http://www.pixijs.com/    ♥♥♥ 


/Users/clarkben/working/RapiD/node_modules/@pixi/utils/src/browser/hello.ts:45
Uncaught TypeError TypeError: rings[0].forEach is not a function
    at <anonymous> (/Users/clarkben/working/RapiD/modules/pixi/PixiFeatureMultipolygon.js:450:16)
    at set geometry (/Users/clarkben/working/RapiD/modules/pixi/PixiFeatureMultipolygon.js:448:9)
    at <anonymous> (/Users/clarkben/working/RapiD/modules/pixi/PixiLayerOsm.js:375:17)
    at renderLines (/Users/clarkben/working/RapiD/modules/pixi/PixiLayerOsm.js:352:14)
    at render (/Users/clarkben/working/RapiD/modules/pixi/PixiLayerOsm.js:247:12)
    at render (/Users/clarkben/working/RapiD/modules/pixi/PixiScene.js:104:13)
    at app (/Users/clarkben/working/RapiD/modules/pixi/PixiRenderer.js:267:16)
    at tick (/Users/clarkben/working/RapiD/modules/pixi/PixiRenderer.js:217:12)
    at TickerListener2.emit (/Users/clarkben/working/RapiD/node_modules/@pixi/ticker/src/TickerListener.ts:67:25)
    at Ticker2.update (/Users/clarkben/working/RapiD/node_modules/@pixi/ticker/src/Ticker.ts:422:37)
    ```

**Comment by Bonkles** [comment]
Removing the 'building' tag from a polygon converts it into a line- which is handled in a separate renderer. So, we need to ensure that we cull any buildings from the map renderer when they lose their 'building-ness' when the tag is removed.

**Comment by Bonkles** [comment]
Fix went in commit `24d8a51ff84faf665f713466345001412988b6ea` and  is deployed to the rapid v2 alpha page. Was successfully able to remove the building tag from a polygon and observe it change to a line.
